sdk/identity/azure-identity/azure/identity/_authn_client.py

```diff
@@ -65,17 +65,32 @@ def __init__(self, endpoint=None, authority=None, tenant=None, **kwargs):  # pyl
             authority = normalize_authority(authority) if authority else get_default_authority()
             self._auth_url = "/".join((authority, tenant.strip("/"), "oauth2/v2.0/token"))
         self._cache = kwargs.get("cache") or TokenCache()  # type: TokenCache
+        self._token_refresh_retry_timeout = DEFAULT_TOKEN_REFRESH_RETRY_TIMEOUT
```

Comment:
"retry_timout" doesn't really describe the usage of this very well, and it makes it harder to understand the code. perhaps `_token_refresh_retry_rate` or `_token_refresh_retry_delay`

Reply:
Updated to use _token_refresh_retry_delay